app/connectors/connectors/sql/sql-common/pom.xml

```diff
@@ -55,8 +55,9 @@
       <scope>provided</scope>
     </dependency>
     <dependency>
-      <groupId>org.json</groupId>
-      <artifactId>json</artifactId>
+      <groupId>com.vaadin.external.google</groupId>
+      <artifactId>android-json</artifactId>
+      <version>0.0.20131108.vaadin1</version>
```

Comment:
Minor thing, can't we use jackson ?

Reply:
+1

Reply:
I can rewrite this to use Jackson, not an issue, let me do that.

Reply:
thx, not a blocking thing